### DISCUSSION TITLE
Constraints python version format

### DISCUSSION THREAD

**Discussion by wwuck** [renovatebot/renovate#19871] [Request Help] [answered]
I saw there was a recent renovate release with this change: https://github.com/renovatebot/renovate/pull/19855

The [pip-compile manager docs](https://docs.renovatebot.com/modules/manager/pip-compile/#configuration-of-python-version) now says to use `"python": "==3.7"` but the [constraints docs](https://docs.renovatebot.com/configuration-options/#constraints) still shows `"python": "2.7"`.  Are they both correct for different situations, or is the second one missing the updated change?

**Comment by rarkins** [comment, marked as answer]
No code has changed. If it works for you then leave it as-is. How/where are you using constraints?

**Reply by wwuck** [reply]
Hmm in that case I might try just removing the setting entirely.  Not really sure why we added it in initially.  Our custom renovate slim docker image installs a single python version via buildpack so I doubt there would be any other python version there usable by renovate anyway, unless there are other versions pulled in as sidecar images somehow?

We are not yet using the pip-compile manager but will test it out next week.

**Comment by JuampiLopez19** [comment]
Hey! It is working for you?
